recipes-demos/ti-apps-launcher/ti-apps-launcher.bb

```diff
@@ -7,17 +7,18 @@ COMPATIBLE_MACHINE = "am62xx|am62pxx|j721s2|j784s4|j722s"
 
 PACKAGE_ARCH = "${MACHINE_ARCH}"
 
-LICENSE = "TI-TFL"
-LIC_FILES_CHKSUM = "file://${WORKDIR}/git/LICENSE;md5=5c3a7f5f6886ba6f33ec3d214dc7ab4c"
+LICENSE = "GPL-2.0-only"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/git/LICENSE;md5=b234ee4d69f5fce4486a80fdaf4a4263"
 
-DEPENDS = "qtbase qtquick3d qtmultimedia"
-RDEPENDS:${PN} = "qtquick3d qtmultimedia bash seva-launcher pulseaudio-service"
+DEPENDS = "qtbase qtquick3d qtdeclarative qtgraphicaleffects qtmultimedia qtxmlpatterns"
+RDEPENDS:${PN} = "qtquick3d qtmultimedia bash seva-launcher pulseaudio-service qtdeclarative-qmlplugins qtgraphicaleffects-qmlplugins"
+RDEPENDS:${PN} = "qtquick3d qtmultimedia bash seva-launcher pulseaudio-service "
```

Comment:
@sskartheekadivi Any specific reasons to add `RDEPENDS` twice? Typo?

Reply:
yup. I forgot 😝 
let me fix it right now.